adoc/admin-troubleshooting.adoc

```diff
@@ -45,4 +45,43 @@ cluster node it was created on.
 After opening a Service Request (SR), you can upload the TAR archives to Global Technical Support.
 
 The data will help to locate the issue you reported and assist you in solving the problem.
-For details, see https://www.suse.com/documentation/sles-12/book_sle_admin/data/cha_adm_support.html.
+For details, see https://www.suse.com/documentation/sles-15/book_sle_admin/data/cha_adm_support.html.
+
+== Debugging Cluster Deployment Failed
+
+If the cluster deployment fails, please re-run the command again with setting verbosity level to 1 `-v=1`.
+
+For example, if bootstraps the first master node of the cluster fails, re-run the command like
+[source,bash]
+----
+skuba node bootstrap --user sles --sudo --target <IP/FQDN> <NODE NAME> -v=1
+----
+
+== Network Planning And Reconfiguration
+
+It is not possible to reconfigure the used IP ranges of SUSE CaaS Platform once the deployment is complete. Therefore, carefully plan for required IP ranges and future scenarios.
+
+Additionally, keep the network requirements in mind, as stated in Deployment Guide Networking chapter.
```

Comment:
@nkoranova I don't know how to link the Deployment Guide Networking chapter.

Reply:
> @nkoranova I don't know how to link the Deployment Guide Networking chapter.

At the moment there is no great way to do that. We can't link between guides. I just write this: `refer to the _{productname} Deployment Guide, Networking chapter_.`.

We have a documentation tool chain feature on the way, that will make this possible though.